README.md

```diff
@@ -9,8 +9,23 @@ discovery sprint with the Department of Justice.
 
 ## Development
 
+Install the dependencies.
+
+    $ gem install bundler
+    $ bundle install
```

Comment:
Thanks all of this Jekyll-izing work @adborden! Was running through these instructions and got the following message:
```
adb-jekyll $ bundle install                                          [11:12:04]
Fetching gem metadata from https://rubygems.org/..........
Fetching version metadata from https://rubygems.org/..
Fetching dependency metadata from https://rubygems.org/.
Resolving dependencies...
activesupport-5.1.2 requires ruby version >= 2.2.2, which is incompatible with
the current version, ruby 2.2.0p0
```

I'm sure I just need to make some updates (I run ruby stuff mayyybe once a year). Do you think it's worth adding some high-level notes about what technologies and versions we're using + any prerequisites or assumptions we have for the local dev instructions?

For example: https://github.com/fedspendingtransparency/usaspending-api/blob/dev/README.md#getting-started